js/classes/entities/midground/weapon/broccoli.js

```diff
@@ -17,7 +17,8 @@ class Broccoli {
             Broccoli.INITIAL_SPEED,
             Broccoli.WEIGHT,
             (block) => this.onBlockCollision(block),
-            (enemy) => this.onEnemyCollision(enemy)
+            (enemy) => this.onEnemyCollision(enemy),
```

Comment:
What is this syntax doing? Is this some sort of arrow function? I'm not super familiar with this in JS being in a parameter list.